notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you change this back to a code cell? (After clicking the cell, either a) look for the list above the notebook that says "Raw" and select "Code" instead, b) press <code>Esc</code> and then <code>Y</code>.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Should I add a specific local path for CRDS and the others? Shall I just use "./"?

Reply:
Hi @camipacifici, I recommend commenting out all of the environment variables with blank strings for now. I'll see how that works with the automated build once the review is over.

Reply:
Done!